pkg/gcv/configs/config.go

```diff
@@ -94,6 +99,23 @@ func asConstraint(data *UnclassifiedConfig) (*Constraint, error) {
 	}, nil
 }
 
+// AsProto returns the constraint a Kubernetes proto
+func (c *Constraint) AsProto() (*validator.Constraint, error) {
+	ci, err := c.Confg.AsInterface()
+	if err != nil {
+		return nil, errors.Wrap(err, "converting to proto")
```

Comment:
converting bytes to go interface?

Reply:
That would be the error returned by `Confg.AsInterface`, this appends that the error is in converting to proto.

Reply:
fine by me